aries-site/src/pages/components/menu.js

```diff
@@ -22,22 +28,147 @@ const Menu = () => {
         <Subsection name={title} level={1} topic={topic}>
           <SubsectionText>{page.description}</SubsectionText>
           <Example
-            code="https://raw.githubusercontent.com/hpe-design/design-system/master/aries-site/src/examples/components/controls/MenuExample.js"
-            details={[
-              <Text size="large">
-                Menu is used to filter or sort content on a page. It is, in some
-                ways, similar to the{' '}
-                <Anchor href="input#select">Select component</Anchor>. However,
-                use the select component when the user must specify from a list
-                of options and submit.
-              </Text>,
-            ]}
+            code="https://raw.githubusercontent.com/hpe-design/design-system/master/aries-site/src/examples/components/menu/MenuExample.js"
+            designer="https://designer.grommet.io/menu?id=HPE-design-system-hpedesignsystem-hpe-com"
             docs="https://v2.grommet.io/menu?theme=hpe#props"
+            figma="https://www.figma.com/file/DyFWlzLBUtK5KH5jJhwoCB/HPE-Menu-Component"
```

Comment:
Proposing we link to the 'thumbnail' portion of the file unless the situation calls for linking to a specific execution
```suggestion
            figma="https://www.figma.com/file/DyFWlzLBUtK5KH5jJhwoCB/HPE-Menu-Component?node-id=38%3A40"
```

Reply:
Yes, sounds good. I just found a thread in Slack that discusses that. I thought the conclusion had been not to link to specific nodes but it was that if a thumbnail is present, we should link to it.